### PR TITLE
implemented CRUD in app

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Screenshot (224)
Screenshot (225)

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Overall code is good
UI also looks nice
PR is good to merge

File: src/Components/Note/Note.jsx [hunk 1]
```diff
@@ -1,15 +1,56 @@
...
+  const { dispatch } = useData();
+  const [edit, setEdit] = useState(false);
+  const { title, content, timeCreated, backgroundColor, _id } = ele;
+
+  const editHandler = () => {
+    setEdit(true);
+  };
+
+  const deleteHandler = async () => {
+    console.log(_id);
```
Comment (ramalingaa) [inline src/Components/Note/Note.jsx:18]: Try removing consoles

**Review by ramalingaa — APPROVED** [review]
Try using one css unit and remove consoles before merging
Code looks clean

**Review by Rohanmond — APPROVED** [review]
looks good to merge

**Pull request merged**